cli/ssh.go

```diff
@@ -354,18 +351,11 @@ func (r *RootCmd) ssh() *clibase.Cmd {
 			Value:       clibase.BoolOf(&noWait),
 		},
 		{
-			Flag:        "log-dir",
-			Default:     os.TempDir(),
-			Description: "Specify the location for the log files.",
-			Env:         "CODER_SSH_LOG_DIR",
-			Value:       clibase.StringOf(&logDir),
-		},
-		{
-			Flag:          "log-to-file",
+			Flag:          "log-file",
```

Comment:
this makes the UX simpler, but harder to use IMO, since the user has to decide where to log.

I held off making logging default-on in my original PR, but if they show value in field debugging, I'd eventually like to make them default-on, which isn't really compatible with making the user choose the filename.